### ISSUE TITLE
Setting `testAllContracts` not working properly

### ISSUE THREAD

**Issue by rappie** [crytic/medusa#250]
My config:
```json
		"deploymentOrder": ["Echidna"],

...

			"testAllContracts": false,
```

With this I still get calls to contracts deployed by `Echidna`. Example:
```
1) ERC20PresetFixedSupply.burn(224) (block=62, time=84, gas=12500000, gasprice=1, value=0, sender=0x0000000000000000000000000000000000020000)
```

**Comment by anishnaik** [comment]
Currently, `testAllContracts` does not work like the multi-abi capability like echidna's. Medusa will still test all dynamically deployed contracts. We need to push a PR through that prevents this behavior. What you can use in the meantime to turn this off is to build off of the `dev/no-multi-abi` branch

https://github.com/crytic/medusa/tree/dev/no-multi-abi

**Comment by rappie** [comment]
Thank you :slightly_smiling_face:

**Comment by rappie** [comment]
It works.

For anyone else using this approach, you also need to add `targetContracts` to your config:

```
		"deploymentOrder": ["Echidna"],
		"targetContracts": ["Echidna"],
```

**Comment by anishnaik** [comment]
Closed by #253
